docs/pydoc/config/invocation-layers.yml

```diff
@@ -37,4 +37,4 @@ renderer:
     descriptive_module_title: true
     add_method_class_prefix: true
     add_member_class_prefix: false
-    filename: invocation_layers_api.md
\ No newline at end of file
+    filename: invocation_layers_api.md
```

Comment:
Is this related to this PR?

Reply:
It's not but I couldn't commit as there was an end-of-the-line fixer issue. See how there is an empty line below, that's why!